x-pack/plugins/ml/public/cases/single_metric_viewer_attachment.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiDescriptionList } from '@elastic/eui';
+import type { PersistableStateAttachmentViewProps } from '@kbn/cases-plugin/public/client/attachment_framework/types';
+import { ReactEmbeddableRenderer } from '@kbn/embeddable-plugin/public';
+import type { FieldFormatsStart } from '@kbn/field-formats-plugin/public';
+import { FIELD_FORMAT_IDS } from '@kbn/field-formats-plugin/common';
+import { FormattedMessage } from '@kbn/i18n-react';
+import deepEqual from 'fast-deep-equal';
+import { memoize } from 'lodash';
+import React from 'react';
+import { CASE_ATTACHMENT_TYPE_ID_SINGLE_METRIC_VIEWER } from '../../common/constants/cases';
+import type {
+  SingleMetricViewerEmbeddableApi,
+  SingleMetricViewerEmbeddableState,
+} from '../embeddables/types';
+
+export const initComponent = memoize((fieldFormats: FieldFormatsStart) => {
+  return React.memo(
+    (props: PersistableStateAttachmentViewProps) => {
+      const { persistableStateAttachmentState, caseData } = props;
+
+      const inputProps =
+        persistableStateAttachmentState as unknown as SingleMetricViewerEmbeddableState;
+
+      const dataFormatter = fieldFormats.deserialize({
+        id: FIELD_FORMAT_IDS.DATE,
+      });
+
+      const listItems = [
+        {
+          title: (
+            <FormattedMessage
+              id="xpack.ml.cases.singleMetricViewer.description.jobIdsLabel"
+              defaultMessage="Job IDs"
```

Comment:
single metric viewer embeddable allows only 1 job 
```suggestion
              defaultMessage="Job ID"
```